iml-agent/src/action_plugins/stratagem/action_filesync.rs

```diff
@@ -0,0 +1,214 @@
+// Copyright (c) 2020 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+extern crate futures;
+extern crate tokio;
+
+use crate::agent_error::{ImlAgentError, RequiredError};
+use crate::lustre::search_rootpath;
+use futures::{
+    future::{self, TryFutureExt},
+    stream::{self, StreamExt, TryStreamExt},
+};
```

Comment:
```suggestion
use futures::{future, stream, StreamExt, TryFutureExt, TryStreamExt};
```

These are all available at the top level.

Reply:
Done